decidim-assemblies/app/commands/decidim/assemblies/admin/update_assemblies_type.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Assemblies
+    module Admin
+      # A command with all the business logic when updating a new assembly
+      # type in the system.
+      class UpdateAssembliesType < Rectify::Command
+        # Public: Initializes the command.
+        #
+        # assemblies_type - A assemblies_type object to update.
+        # form - A form object with the params.
+        def initialize(assemblies_type, form)
+          @assemblies_type = assemblies_type
+          @form = form
+        end
+
+        # Executes the command. Broadcasts these events:
+        #
+        # - :ok when everything is valid.
+        # - :invalid if the form wasn't valid and we couldn't proceed.
+        #
+        # Returns nothing.
+        def call
+          return broadcast(:invalid) if form.invalid?
+
+          update_assemblies_type!
+
+          broadcast(:ok)
+        end
+
+        private
+
+        attr_reader :form
+
+        def update_assemblies_type!
+          @assemblies_type.update!(
```

Comment:
I'm not sure if `Decidim.traceability` is required here, what do you think?

Reply:
I wasn't sure to use traceability in any of the actions, but if we are going to add it in create and destroy we should here too